reco_utils/README.md

```diff
@@ -54,17 +111,39 @@ Currently available metrics include:
 The recommender submodule contains implementations of various algorithms that can be used in addition to external packages to evaluate and develop new recommender system approaches. A description of all the algorithms can be found on [this table](../README.md#algorithms). Next a list of the algorithm utilities:
 
 * Cornac
-* DeepRec (includes xDeepFM and DKN)
+* DeepRec
+  *  Convolutional Sequence Embedding Recommendation (CASER)
+  *  Deep Knowledge-Aware Network (DKN)
+  *  Extreme Deep Factorization Machine (xDeepFM)
+  *  GRU4Rec
+  *  LightGCN
+  *  Next Item Recommendation (NextItNet)
+  *  Short-term and Long-term Preference Integrated Recommender (SLi-Rec)
+  *  Multi-Interest-Aware Sequential User Modeling (SUM)
 * FastAI
+* GeoIMC
+* LightFM
 * LightGBM
 * NCF
-* NewsRec (includes LSTUR, NAML NPA and NRMS)
-* RBM
-* RLRMC
-* SAR
+* NewsRec
+  * Neural Recommendation with Long- and Short-term User Representations (LSTUR)
+  * Neural Recommendation with Attentive Multi-View Learning (NAML)
+  * Neural Recommendation with Personalized Attention (NPA)
+  * Neural Recommendation with Multi-Head Self-Attention (NRMS)
+  * (includes LSTUR, NAML NPA and NRMS)
```

Comment:
`* (includes LSTUR, NAML NPA and NRMS)` not sure I understand this line, there was a description of these algos above

Reply:
right, copy-paste error, will remove